app/models/link_set.rb

```diff
@@ -1,42 +1,26 @@
 class LinkSet < ActiveRecord::Base
-  include Replaceable
-  include DefaultAttributes
-  include SymbolizeJSON
-
-  validate :links_are_valid
+  has_many :links
 
-  def links=(links_hash)
-    super(links_hash || {})
-  end
+  include Replaceable
 
-private
-  def self.query_keys
-    [:content_id]
-  end
+  def hashed_links
```

Comment:
I find this method name quite confusing. I was thinking MD5 hashes.

My understanding is that this method is building a hash of the links. Would `#as_json` suffice? If not, `#to_hash` might be better?